Expensify/App
# Added Spanish translation for example check image and Company types

@iwiznia  PR is ready for review.

### Details
Spanish translation was missing for Company types, and Routing and Bank Account Number example image in VBA flow. So added translation for both.

### Fixed Issues
$ https://github.com/Expensify/App/issues/5992

### Tests | QA Steps
1. Set app to spanish
2. Navigate to bank account flow > Manually set bank account
3. Check the image example above the routing and bank account number
4. Enter test credentials and proceed to company step
5. Click on the company type dropdown

QA Check 1:  Now example check image shows in Spanish Or English as per language preference.
QA Check 2:  Company Types dropdown will be translated as per language preference.

### Tested On


### Screenshots

#### Web
https://user-images.githubusercontent.com/7823358/140092975-6e9d8045-940b-4aba-8e99-664ed494c3c6.mp4

#### Mobile Web
https://user-images.githubusercontent.com/7823358/140093036-f8aa2e90-ab21-41b9-b4e7-b8e8b8c1f03d.mp4

#### Desktop
https://user-images.githubusercontent.com/7823358/140093103-3bc97e98-59c3-4514-b726-d5b93f10bdb7.mp4

#### iOS
https://user-images.githubusercontent.com/7823358/140093177-584894ce-94c5-48b4-b02a-8743c43899c4.mp4

#### Android
https://user-images.githubusercontent.com/7823358/140093255-33d290b4-4613-4545-a112-7183c55d4d88.mp4
